mod/zaudio/views/default/js/zaudio/boot.js

```diff
@@ -0,0 +1,16 @@
+define(function (require) {
+	var $ = require('jquery');
+
+	return {
```

Comment:
A type would mean users have to add another dependency to all boot modules and you still have to check unexpected keys, but it might get still make more sense to users.